website/content/apis/_index.md

```diff
@@ -969,3 +969,4 @@ BGPPeerStatus
 Generated with <code>gen-crd-api-reference-docs</code>
 .
 </em></p>
+dsadasdsa
```

Comment:
Looks like the site build lane caught it, plus the CI lane works as intended.
LGTM!